platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/PolicyManager.java

```diff
@@ -46,9 +47,11 @@
 
     private static final String WHITE_LIST = "whiteListContexts";
 
-    private static final String DEFAULT_REALM = "DDF";
+    public static final String DEFAULT_REALM = "DDF";
 
-    private static final String DEFAULT_REALM_CONTEXTS = "/=karaf";
+    public static final String DEFAULT_REALM_CONTEXT_VALUE = "karaf";
+
+    private static final String DEFAULT_REALM_CONTEXTS = "/=" + DEFAULT_REALM_CONTEXT_VALUE;
 
     private Map<String, ContextPolicy> policyStore = new HashMap<>();
```

Comment:
We are calling ```Collections.unmodifiableCollection(new ArrayList<>(policyStore.values()))``` to set ```Collection<ContextPolicy> currentPolicies```. Although I'm not using currentPolicies in getContextPolicy. Whoops